tests/test_rest.py

```diff
@@ -40,21 +40,29 @@ def test_strip_kwargs():
     assert strip_kwargs(**dict(a=[])) == {"a": []}
 
 
-class TestAnotherBaseModel(BaseModel):
+class AnotherBaseModel(BaseModel):
 
     some_float: float
     some_bool: bool
 
 
-class TestBaseModel(BaseModel):
+class BaseModel(BaseModel):
```

Comment:
Perhaps ExampleBaseModel since BaseModel is already used by pydantic
```suggestion
class ExampleBaseModel(BaseModel):
```